recipe/meta.yaml

```diff
@@ -80,105 +73,190 @@ requirements:
+        {% set tests_to_skip = tests_to_skip + " or (test_lsq_linear and test_large_rank_deficient)" %}   # [aarch64 or ppc64le]
+        {% set tests_to_skip = tests_to_skip + " or (test_iterative and test_precond_inverse[case1])" %}  # [aarch64 or ppc64le]
+        {% set tests_to_skip = tests_to_skip + " or (test_optimize and TestBrute and test_workers)" %}    # [aarch64 or ppc64le]
+        # tests that run into timeouts (set in scipy test suite) in emulation
+        {% set tests_to_skip = tests_to_skip + " or (test_propack and test_examples)" %}                  # [aarch64 or ppc64le]
+        # occasionally runs out of memory on windows + pypy
+        {% set tests_to_skip = tests_to_skip + " or test_bootstrap_against_" %}                           # [(python_impl == "pypy") and win]
+        # very slow tests for pypy in emulation
+        {% set pypy_emu = (python_impl == "pypy") and (aarch64 or ppc64le) %}
+        {% set tests_to_skip = tests_to_skip + " or TestDualAnnealing or TestQuad or TestNQuad" %}        # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or TestDifferentialEvolutionSolver or TestTRF" %}        # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or TestMonteCarloHypothesisTest" %}                      # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or (TestBessel and test_fs_param)" %}                    # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or (TestOrdQZWorkspaceSize and test_decompose)" %}       # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_all_nograd_minimizers or test_basic_fit_mle" %}  # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_cdf_against_tables or test_pdf_integration" %}   # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_cdist_ and (refcount or calling_convention)" %}  # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_conditionally_positive_definite" %}              # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_cont_basic[500 or test_cython_api[elliprj]" %}   # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_filtfilt_gust or test_kappa3_array_gh13582" %}   # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_fitstart[genexpon-shapes28] or test_gh12922" %}  # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_integration_sparse or test_kdtree_box" %}        # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_minimize_callback_copies_array[fmin]" %}         # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_mip1 or test_mixed_threads_processes" %}         # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_modules_importable or test_neldermead_limit" %}  # [pypy_emu]
+        {% set tests_to_skip = tests_to_skip + " or test_random_exact or test_random_complex_exact" %}    # [pypy_emu]
```

Comment:
Some of these shouldn't be running I think because of the `set label = "'fast'"` logic a few lines below. For the ones that are actually needed, how about we upstream the skips?

Reply:
All these are current (as in: running even with `label='fast'`). I updated them recently in https://github.com/conda-forge/scipy-feedstock/pull/227